src/views/AnnualReport.vue

```diff
@@ -674,6 +674,8 @@ export default {
           }
         }
 
+        // NB: ARFilingYear, arMaxDate and arMinDate were set in the store by the TodoList component
```

Comment:
This seems kind of weird but is needed when starting a new AR.

A better way might be for the dashboard to create a draft AR with the initial data, and then THIS component could always load the draft and have the data it needs... future (if ever).

Reply:
Another way would be for the task object to be passed to the Annual Report component. FWIW